api/caasfirewaller/client.go

```diff
@@ -89,7 +89,7 @@ func (c *Client) Life(appName string) (life.Value, error) {
 	if err := results.Results[0].Error; err != nil {
 		return "", maybeNotFound(err)
 	}
-	return life.Value(results.Results[0].Life), nil
+	return results.Results[0].Life, nil
```

Comment:
Is this because later Go versions were better at similar type assignments? I'm assuming this is the case, and not a change in either underlying type.

Reply:
No this is because the unconvert linter was never running properly, so these were supposed to be picked up before.